var/spack/repos/builtin/packages/salmon/package.py

```diff
@@ -55,10 +56,18 @@ class Salmon(CMakePackage):
     depends_on("libgff")
     depends_on("pkgconfig")
     depends_on("curl", when="@0.14.1:")
+    depends_on("htslib", when="@1.10.2")
```

Comment:
I'm assuming this'll be a requirement going forward, so probably more appropriate for this to be when="@1.10.2:"